<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>821</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>110</y>
     <width>201</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>87</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(162, 255, 41);</string>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>自增</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>自减</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcdNumber"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>726</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>red()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>727</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1()</signal>
  <slot>add()</slot>
  <slot>red()</slot>
  <slot>slot1()</slot>
 </slots>
</ui>
